packages/core/src/components/Tabs/Tab/__tests__/Tab.test.tsx

```diff
@@ -1,17 +1,67 @@
 import React from "react";
-import { render, fireEvent, screen } from "@testing-library/react";
-import Tab from "../Tab";
+import { render, fireEvent, screen, cleanup } from "@testing-library/react";
+import { act } from "@testing-library/react-hooks";
+import Tab, { TabProps } from "../Tab";
+
+const onClickMock = jest.fn();
+const text = "tab";
+
+const renderComponent = (props: TabProps = {}) => {
+  return render(
+    <Tab onClick={onClickMock} {...props}>
+      {text}
+    </Tab>
+  );
+};
 
 describe("Tab tests", () => {
-  const onClickMock = jest.fn();
-  const text = "tab";
+  afterEach(() => {
+    cleanup();
+  });
 
-  beforeEach(() => {
-    render(<Tab onClick={onClickMock}>{text}</Tab>);
+  describe("click", () => {
+    it("should call the click callback when clicked", () => {
+      renderComponent();
+      fireEvent.click(screen.getByText(text));
+      expect(onClickMock.mock.calls.length).toBe(1);
+    });
   });
 
-  it("should call the click callback when clicked", () => {
-    fireEvent.click(screen.getByText(text));
-    expect(onClickMock.mock.calls.length).toBe(1);
+  describe("Tooltips", () => {
+    it("should not display tooltip content from tooltipProps if disabled is false", () => {
```

Comment:
This will be redundant when we change the tooltipProps condition of the disabled